src/AddressCard.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import { bonds, InlineAccount, InlineBalance, AccountIcon, AddressLabel, AccountLabel, EtherBalance, CoinList } from 'parity-reactive-ui';
+import { Header, Card, Image, Table, Button, Popup, Input } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import { DappLink } from '@parity/ui';
+import { BigNumber } from 'bignumber.js';
+import { Bond } from 'oo7';
+import { formatBalance } from 'oo7-parity';
+import { ReactiveComponent, Rspan, Rimg } from 'oo7-react';
+import _ from 'lodash';
+
+import styles from './AddressCard.css';
+
+//TODO: Add this to semantic-ui-react
+export default class AddressCard extends Component{
```

Comment:
This really should be run through the linter, I'm assuming it is not enabled in your IDE. (Pushing changes to allow for `npm run lint` but if the editor is configured properly it should just see there is .eslintrc and show the issues in the code - I see a lot where we are not aligned in terms of eslint styles)